src/s1ap/handlers/s1ap_msg_delegator.c

```diff
@@ -468,6 +473,11 @@ int convertUplinkNasToProtoIe(InitiatingMessage_t *msg, struct proto_IE* proto_i
 							return -1;
 						}
 
+                        if (s1apNASPDU_p->size > 300) {
```

Comment:
same comment - how you arrived at 300?

Reply:
I relied on the same assumption as above, which may be wrong. In this case, `s1Msg` is of type `uplink_nas_t`, which is defined here:

https://github.com/omec-project/Nucleus/blob/a8002eb10e10554593ebd5a94ec2d3fdb8ee467a/include/common/msgType.h#L616-L631

It likewise uses a `rawNas_Q_msg_t`:

https://github.com/omec-project/Nucleus/blob/a8002eb10e10554593ebd5a94ec2d3fdb8ee467a/include/common/msgType.h#L114-L118

So yes, it should be changed to `MAX_NAS_MSG_SIZE` as well. Ill get this fixed too.

Reply:
I relied on the same assumption as above, which may be wrong. In this case, `s1Msg` is of type `uplink_nas_t`, which is defined here:

https://github.com/omec-project/Nucleus/blob/a8002eb10e10554593ebd5a94ec2d3fdb8ee467a/include/common/msgType.h#L616-L631

It likewise uses a `rawNas_Q_msg_t`:

https://github.com/omec-project/Nucleus/blob/a8002eb10e10554593ebd5a94ec2d3fdb8ee467a/include/common/msgType.h#L114-L118

So yes, it should be changed to `MAX_NAS_MSG_SIZE` as well. Ill get this fixed too.